src/components/Footer.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react"
+
+const Footer = () => {
+  return (
+    <footer>
+      <div className="footer-div" id="github-link-container">
```

Comment:
Please remove the id on the two elements as they are not used.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
removed ✔

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good